gnd/src/test/java/com/google/android/gnd/persistence/local/LocalDataStoreTest.java

```diff
@@ -67,97 +66,138 @@
 @Config(application = TestApplication.class)
 public class LocalDataStoreTest {
 
+  private static final User TEST_USER =
+      User.builder().setId("user id").setEmail("user@gmail.com").setDisplayName("user 1").build();
+
+  private static final MultipleChoice TEST_MULTIPLE_CHOICE =
+      MultipleChoice.newBuilder()
+          .setCardinality(Cardinality.SELECT_ONE)
+          .setOptions(
+              ImmutableList.of(
+                  Option.newBuilder().setCode("a").setLabel("Name").build(),
+                  Option.newBuilder().setCode("b").setLabel("Age").build()))
+          .build();
+
+  private static final Field TEST_FIELD =
+      Field.newBuilder()
+          .setId("field id")
+          .setLabel("field label")
+          .setRequired(false)
+          .setType(Type.MULTIPLE_CHOICE)
+          .setMultipleChoice(TEST_MULTIPLE_CHOICE)
+          .build();
+
+  private static final Form TEST_FORM =
+      Form.newBuilder()
+          .setId("form id")
+          .setElements(ImmutableList.of(Element.ofField(TEST_FIELD)))
+          .build();
+
+  private static final Layer TEST_LAYER =
+      Layer.newBuilder()
+          .setId("layer id")
+          .setItemLabel("item label")
+          .setListHeading("heading title")
+          .setDefaultStyle(Style.builder().setColor("000").build())
+          .setForm(TEST_FORM)
+          .build();
+
+  private static final Project TEST_PROJECT =
+      Project.newBuilder()
+          .setId("project id")
+          .setTitle("project 1")
+          .setDescription("foo description")
+          .putLayer("layer id", TEST_LAYER)
+          .build();
+
+  private static final Point TEST_POINT =
+      Point.newBuilder().setLatitude(110.0).setLongitude(-23.1).build();
+
+  private static final FeatureMutation TEST_FEATURE_MUTATION =
+      FeatureMutation.builder()
+          .setId(1L)
+          .setFeatureId("feature id")
+          .setType(Mutation.Type.CREATE)
+          .setUserId("user id")
+          .setProjectId("project id")
+          .setLayerId("layer id")
+          .setNewLocation(Optional.ofNullable(TEST_POINT))
+          .setClientTimestamp(new Date())
+          .build();
+
+  private static final ObservationMutation TEST_OBSERVATION_MUTATION =
+      ObservationMutation.builder()
+          .setObservationId("observation id")
+          .setType(Mutation.Type.CREATE)
+          .setProjectId("project id")
+          .setFeatureId("feature id")
+          .setLayerId("layer id")
+          .setFormId("form id")
+          .setUserId("user id")
+          .setResponseDeltas(
+              ImmutableList.of(
+                  ResponseDelta.builder()
+                      .setFieldId("field id")
+                      .setNewResponse(TextResponse.fromString("response for field id"))
+                      .build()))
+          .setClientTimestamp(new Date())
+          .build();
+
+  private static final Tile TEST_TILE_PENDING =
```

Comment:
Nit: Please use natural word order for readability: TEST_PENDING_TILE here and below.